Corona-Warn-App/src/main/java/de/rki/coronawarnapp/nearby/modules/version/DefaultENFVersion.kt

```diff
@@ -41,8 +40,6 @@ class DefaultENFVersion @Inject constructor(
     }
 
     override suspend fun isAtLeast(compareVersion: Long): Boolean {
-        if (!compareVersion.isCorrectVersionLength) throw IllegalArgumentException("given version has incorrect length")
```

Comment:
Comparing based on length is not valid anymore. latest version has longer length. I think Long comparison is enough here